pandas/core/internals.py

```diff
@@ -2918,14 +2918,15 @@ def make_block(values, placement, klass=None, ndim=None, dtype=None,
         # GH#19265 pyarrow is passing this
         warnings.warn("fastpath argument is deprecated, will be removed "
                       "in a future release.", DeprecationWarning)
+
     if klass is None:
         dtype = dtype or values.dtype
         klass = get_block_type(values, dtype)
 
     elif klass is DatetimeTZBlock and not is_datetimetz(values):
         return klass(values, ndim=ndim,
                      placement=placement, dtype=dtype)
-
+    
```

Comment:
You might want to install a flake8 plugin for your editor ;)

Reply:
ok ok installing now. I fully admit that I'm awful when it comes to linting failures and should just tool the fix. Might as well install Ctrl-P while I'm at it.